doc/service-proxy.md

```diff
@@ -105,11 +125,12 @@ to ensure that it does. Eg:
     reply directly to itself, rather than sending it back to the host
     network namespace for un-DNAT'ing first.
 
-  - Likewise, if a host-network process tries to connect to a NodePort
-    service via `127.0.0.1` or `::1`, it is necessary to SNAT the
-    packet, since otherwise the destination pod would think the packet
-    came from its own `127.0.0.1`/`::1`. (In fact, the kernel will log
-    a warning if you try to DNAT-but-not-SNAT a packet whose source IP
+  - Likewise, if the proxy supports localhost NodePort connections,
+    and a host-network process tries to connect to a NodePort service
+    via `127.0.0.1` or `::1`, it is necessary to SNAT the packet,
```

Comment:
this is the route_localnet hack, no? 

this doesn't work for IPv6 no matter what https://github.com/kubernetes/kubernetes/issues/90236

Reply:
Ah! I noticed that this contradicted [the comments in the kube-proxy source code](https://github.com/kubernetes/kubernetes/blob/8f94681cd294aa8cfd3407b8191f6c70214973a4/pkg/proxy/iptables/proxier.go#L1437-L1439) but I was thinking this doc was right and the code was wrong.